.github/workflows/ci-workflow.yml

```diff
@@ -69,22 +73,25 @@ jobs:
       - name: Clone
         uses: actions/checkout@v2
 
-      - name: Build
+      - name: Build Exchange
         run: |
-          make BOLOS_SDK=$NANOSP_SDK DEBUG=1 TESTING=1 TEST_PUBLIC_KEY=1
-          cp bin/app.elf exchange_nanosp.elf
+          make clean ; BOLOS_SDK=${{ matrix.sdk.path }} TESTING=1 TEST_PUBLIC_KEY=1 make
```

Comment:
Setting the variables before the make call can run into issues sometimes (not here though), as Ledger Makefiles often use ':=' instead of '?=' for conditional assignation
We talked about this a bit with Alex, maybe we should do a quick meeting about this and decide on the way to go

Reply:
Never though of this, I'm used to put them after.